docs/src/humann.md

```diff
@@ -1,4 +1,613 @@
+
+If any of those things aren't true, or you don't know if they're true,
+go back to ["Getting Started"](@ref Getting-Started) to see if you skipped a step.
+If you're still confused, please ask (see 3rd bullet point at the top)!
+
+### HUMAnN Databases
+
+HUMAnN requires a number of specialized databases to work correctly.
+When you first install it, it comes with some demo databases that are much smaller,
+but can be used to complete this tutorial.
+However, for actually running real data, you'll want to take the time
+to download them - they're BIG!
+[See here](https://github.com/biobakery/humann#5-download-the-databases) for more information.
+
+For now, the easiest way to do this for now is via the shell,
+which you can access from the julia REPL by typing `;`:
+
+```julia-repl
+shell> humann_databases --help
+usage: humann_databases [-h] [--available]
+                        [--download <database> <build> <install_location>]
+                        [--update-config {yes,no}]
+                        [--database-location DATABASE_LOCATION]
+
+HUMAnN Databases
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --available           print the available databases
+  --download <database> <build> <install_location>
+                        download the selected database to the install location
+  --update-config {yes,no}
+                        update the config file to set the new database as the default [DEFAULT: yes]
+  --database-location DATABASE_LOCATION
+                        location (local or remote) to pull the database
+
+shell> humann_databases --available
+HUMANnN2 Databases ( database : build = location )
+chocophlan : full = http://huttenhower.sph.harvard.edu/humann_data/chocophlan/full_chocophlan.v296_201901b.tar.gz
+chocophlan : DEMO = http://huttenhower.sph.harvard.edu/humann_data/chocophlan/DEMO_chocophlan.v296_201901b.tar.gz
+uniref : uniref50_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref50_annotated_v201901b_ful
+l.tar.gz
+uniref : uniref90_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref90_annotated_v201901b_ful
+l.tar.gz
+uniref : uniref50_ec_filtered_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_ec_filtered/uniref50_ec_filte
+red_201901b_subset.tar.gz
+uniref : uniref90_ec_filtered_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_ec_filtered/uniref90_ec_filte
+red_201901b_subset.tar.gz
+uniref : DEMO_diamond = http://huttenhower.sph.harvard.edu/humann_data/uniprot/uniref_annotated/uniref90_DEMO_diamond_v201901b.tar.
+gz
+utility_mapping : full = http://huttenhower.sph.harvard.edu/humann_data/full_mapping_v201901b.tar.gz
+```
+
+For example, if you'd like to install these databases to `/BigDrive/humann/`,
+you could run
+
+```julia-repl
+shell> humann_databases --download cholophlan full /BigDrive/humann/chocophlan
```

Comment:
I'm not sure this is anything we can fix since this is in humann, but when I run this command, I get 
```
Creating directory to install database: /BigDrive/humann/chocophlan
CRITICAL ERROR: Unable to create directory: /BigDrive/humann/chocophlan
```
I'm not sure how to get this to work.